lib/novelconverter.rb

```diff
@@ -30,6 +32,33 @@ class NovelConverter
   end
 
   #
+  # Get extensions for each device
+  #
+  def self.get_txt_ext
```

Comment:
__[RuboCop]__

Naming/AccessorMethodName: Do not prefix reader method names with `get_`.


<!-- Comment by [SideCI](https://sideci.com) -->